pages/components/Welcome.tsx

```diff
@@ -0,0 +1,20 @@
+import useSWR from 'swr';
+import { withPageAuthRequired } from '@auth0/nextjs-auth0';
+
+
+const fetcher = async (uri: RequestInfo) => {
+  const response = await fetch(uri);
+  return response.json();
+};
+
+export default withPageAuthRequired(function Welcome() {
```

Comment:
If possible, please use here an arrow function instead of a function. It looks slightly odd defining a function with a name and than not referencing it.